src/Suave/Combinators.fs

```diff
@@ -888,7 +891,8 @@ module CORS =
               >=> setAllowMethodsHeader config "*"
             composed ctx
           else
-            succeed ctx // No headers will be sent. Browser will deny.
+            if (config.checkWebSocketOrigin && (req.header "upgrade"  |> Choice.map (fun s -> s.ToLower()) = Choice1Of2 "websocket")) then fail
```

Comment:
Extra outer parenthesis not needed in if statement.

The `String` module contains the toLowerInvariant method that may be clearer and also doesn't go through the current CultureInfo.